source/administration/devops-command-center.rst

```diff
@@ -0,0 +1,268 @@
+  :backlinks: top
+  :local:
+  :depth: 2
+
+Overview
+^^^^^^^
+
+Workplace messaging is often process-driven rather than completely ad-hoc, and these processes could be organizationally critical in crisis situations. Help teams collaborate to solve problems that are time-sensitive, recurring, situational.
+
+Incident Response is a Mattermost plugin designed to help organizations monitor, coordinate, and measure their incident response processes, increasing transparency, maximizing effectiveness, and ultimately saving costs by cutting down time taken to respond and resolve incidents.
+
+
+User's Guide
+^^^^^^^^^^^^^
+
+Incidents are events within Mattermost that are initiated to manage a specific situation/response in real-time. When the situation/response is complete, the incident is ended. Incident details are stored as logs, and events that occurred within the incident can be reused in future incidents as Playbooks and checklists.
+
+Starting incidents
+~~~~~~~~~~~~~~~~~~
+
+The incident channel is the central place for discussion related to the incident. You can start an incident in one of three ways:
+
+- Use the slash command */incident start* from any channel.
+- Use the message actions dropdown menu and select **Start incident**.
+- Use the context menu of a post and select **Start incident**.
+
+When you create an incident, the name provided is applied to the new incident channel that is created. The channel name can be used to search for the channel, and is also the name appended to the archived logs. You can also select an optional Playbook. If a Playbook is selected, the corresponding checklist is used as a template to start the incident. If no Playbook is selected, the incident starts with an empty checklist.
+
+The creator of an incident automatically becomes the incident Commander, responsible for managing the incident.
+
+The Commander of the incident can:
+
+- Select a Playbook when starting an incident
+- Create checklists
+- Invite people to the channel
+- Hand over the Commander role to another channel member
+
+To change Commanders, select the current Commander’s name and use the search bar to locate the username of the new Commander. Only members of the channel may be selected as commanders. To assign a Commander who is not in the channel, that user must first be invited to the channel using the existing Mattermost user interface (**Manage Members** in the channel header, or the */invite* slash command). Changing the Commander takes effect immediately, and members are notified of the change in the channel.
+
+Active incidents
+~~~~~~~~~~~~~~~~
+
+To view details about active incidents, select **! Incidents** from the plugin menu in the channel header to open the right-hand side (RHS) panel and list of current active incidents. Select an incident to see its Commander, channel, and checklist.
+
+When an incident has started and the incident channel is created, the channel header is updated to reflect its origin and
```

Comment:
nit: This is obviously from the technical design, but is this the level of detail we want in the manual? Wondering if a breakdown of the functionality is the right approach to document this for an enduser.